subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ConcurrentBuildsArtifactTransformIntegrationTest.groovy

```diff
@@ -17,14 +17,20 @@
 package org.gradle.integtests.resolve.transform
 
 import org.gradle.integtests.fixtures.AbstractDependencyResolutionTest
+import org.gradle.integtests.fixtures.IncrementalArtifactTransformationsRunner
 import org.gradle.test.fixtures.server.http.BlockingHttpServer
 import org.junit.Rule
+import org.junit.runner.RunWith
 
+import static org.gradle.integtests.fixtures.IncrementalArtifactTransformationsRunner.configureIncrementalArtifactTransformations
+
+@RunWith(IncrementalArtifactTransformationsRunner)
 class ConcurrentBuildsArtifactTransformIntegrationTest extends AbstractDependencyResolutionTest {
     @Rule BlockingHttpServer server = new BlockingHttpServer()
 
     def setup() {
         server.start()
+        configureIncrementalArtifactTransformations(settingsFile)
```

Comment:
Do we need this everywhere so the tests are executed both with incremental off and on?

Reply:
Yes. The Runner sets the static field on itself and executes each test with the field set to true/false. We still need to react to the field in the actual test - this is what we do here.

Reply:
Should the runner be called `Experimental...Runner`? That would capture the fact that it will go away as soon as the feature flag is gone.

Reply:
I renamed the runner to `Experimental...`